packages/salesforcedx-vscode-soql/package.json

```diff
@@ -127,6 +127,11 @@
       {
         "command": "soql.builder.open.new",
         "title": "%soql_builder_open_new%"
+      },
+      {
+        "command": "soql.builder.toggle",
+        "title": "%soql_builder_toggle%",
+        "icon": "$(go-to-file)"
```

Comment:
How did you find this icon? Does VS Code give us access to all the default icons they use?

Reply:
Yes, i found it here:  https://code.visualstudio.com/api/references/icons-in-labels

Apparently, they process this with something kinda like handebars and look for icons.  They don't specifically mention it, but based on some stack overflow questions and github issues, I was able to piece this implementation together.  And it worked!

Reply:
thats pretty neat!